docs/core/extensions/globalization-icu.md

```diff
@@ -26,6 +26,9 @@ Before .NET 5, the .NET globalization APIs used different underlying libraries o
 
 Starting with .NET 5, developers have more control over which underlying library is used, enabling applications to avoid differences across platforms.
 
+> [!NOTE]
+> The culture data that drives the behavior of the ICU library is usually maintained by the [Common Locale Data Repository (CLDR)](https://cldr.unicode.org/), not by the runtime or the host OS.
```

Comment:
>not by the runtime or the host OS. [](http://example.com/codeflow?start=159&length=34)

Actually this is not accurate as Windows maintain ICU versions which customized a little to fit Windows requirements.  It is possible can run into some data that is not exactly matching CLDR.

Maybe saying `not by the runtime `  and delete the part `or the host OS.` be reasonable?